Source/Core/TranslationRotationScale.js

```diff
@@ -0,0 +1,78 @@
+/*global define*/
+define([
+        './Cartesian3',
+        './defaultValue',
+        './defined',
+        './Matrix4',
+        './Quaternion'
+    ], function(
+        Cartesian3,
+        defaultValue,
+        defined,
+        Matrix4,
+        Quaternion) {
+    "use strict";
+
+    var defaultScale = new Cartesian3(1.0, 1.0, 1.0);
+    var defaultTranslation = Cartesian3.ZERO;
+    var defaultRotation = Quaternion.IDENTITY;
+
+    /**
+     * A set of transformations to apply to a particular node in a {@link Model}.
```

Comment:
This is kind of more generic now, right?  @pjcozzi do you have a suggestion for a one-line definition of TRS?